cg/models/run_devices/pacbio_cell_directory_data.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from cg.constants.pacbio import PacBioDirsAndFiles
+from cg.utils.files import get_files_in_directory_with_pattern
+
+
+class PacBioCellDirectoryData:
+    """Class to collect information about PacBio sequencing run directories."""
+
+    def __init__(self, pac_bio_run_directory: Path):
+        self.path = pac_bio_run_directory
+        self.statistics_dir = Path(self.path, "statistics")
+
+    @property
+    def ccs_report(self) -> Path:
+        """Finds the json file finishing with ccs_report.json in the statistics dir."""
+        files: list[Path] = get_files_in_directory_with_pattern(
+            directory=self.statistics_dir, pattern=PacBioDirsAndFiles.CCS_REPORT_SUFFIX
+        )
+        return files[0]
```

Comment:
I think if we start putting the files as property on this model we will end up with the same that happened with the IlluminaRunDirectoryData, which is totally overloaded and lost its single responsibility.

So what should this class achieve? It should collect information but here in this property it returns a file. Should it not just return the expected path? And then getting the file should be done somewhere else?

Reply:
Fixing in pair programming session